microprofile/6.1/telemetry/1.1/23.0.0.12-MicroProfile-Telemetry-1.1-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,259 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Telemetry 1.1
+
+== Open Liberty 23.0.0.12 - MicroProfile Telemetry 1.1 Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/23.0.0.12/openliberty-23.0.0.12.zip[Open Liberty 23.0.0.12]
+
+* Specification Name, Version and download URL:
++
+https://github.com/eclipse/microprofile-telemetry/tree/1.1[MicroProfile Telemetry 1.1 Specification]
+
+* (Optional) TCK Version, digital fingerprint and download URL:
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/telemetry/tracing/microprofile-telemetry-tracing-tck/1.1/microprofile-telemetry-tracing-tck-1.1.jar[MicroProfile Telemetry 1.1 TCK]
++
+SHA-1: `c4ef66abc825116acdf28db8fe55107461ee1395`
++
+SHA-256: `e3104d1703e1ff2f631c7e6e32d6bfd78c44e046f3f1452f6da1d3eb1787beda`
+
+* Public URL of TCK Results Summary:
++
+xref:23.0.0.12-MicroProfile-Telemetry-1.1-Java11-TCKResults.adoc[TCK results summary]
+
+
+* Java runtime used to run the implementation:
++
+Java 11: OpenJDK Runtime Environment (11.0.21+9)
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+Mac OS X: kern.version: Darwin Kernel Version 23.0.0: Fri Sep 15 14:42:42 PDT 2023; root:xnu-10002.1.13~1/RELEASE_X86_64
+
+Test results:
+
+[source, text]
+----
+Test suite: org.eclipse.microprofile.telemetry.tracing.tck.cdi.OpenTelemetryBeanTest 2023-12-01T12:52:09 UTC
+Tests:2 Failures:0 Errors:0
+   org.eclipse.microprofile.telemetry.tracing.tck.cdi.OpenTelemetryBeanTest.testOpenTelemetryBean_EE9_FEATURES_MicroProfile_50_MPTEL11 Passed!
```

Comment:
Is Telemetry supposed to include MP 5 / EE 9 results?  If so, should it include MP 4.1 / EE 8 and MP 1.4 and EE 7 as well?

Reply:
No, should only have MP 6.1. Ideally we need to fix the code that generates these docs so that the results are not mixed together like this.